core/unit_test/TestReduceCombinatorical.hpp

```diff
@@ -553,95 +567,103 @@ struct TestReduceCombinatoricalInstantiation {
   }
 
   template <class... Args>
-  static void AddPolicy_1(Args... args) {
-    int N = 1000;
+  static void AddPolicy_1(int N, Args... args) {
     Kokkos::RangePolicy<ExecSpace> policy(0, N);
 
-    AddFunctorLambdaRange(args..., 1000);
-    AddFunctorLambdaRange(args..., N);
-    AddFunctorLambdaRange(args..., policy);
+    AddFunctorLambdaRange(1000, args..., 1000);
```

Comment:
Should this run repeatedly independently of `N`?
```suggestion
```

Reply:
yeah it doesn't matter much. Otherwise I need to extract that in yet another different function.

Reply:
I.e. I need to add a AddPolicy_1a or so, i think running repeatedly that thing is fine.

Reply:
Hmm... but we call AddPolicy_1 with N=1000 below so this check is the same as the next line, isn't it?

Reply:
this is very explicitly checking literals vs runtime parameters.